cmd/catalog-importer/cmd/VERSION

```diff
@@ -1 +1 @@
-2.6.2
+2.6.3
```

Comment:
with this PR changing default behaviour lets bump the minor (instead of the patch)
```suggestion
2.7.0
```